### DISCUSSION TITLE
elastic constant calculation in ABACUS

### DISCUSSION THREAD

**Discussion by hongriTianqi** [deepmodeling/abacus-develop#683] [Ideas]
## Background
Elastic constants are important to analyse the mechanical properties of solids. ABACUS can calculate stress in optimization, can elastic constants and Young's modulus be given for an optimized structure in ABACUS?

## Describe the solution you'd like
Add the function of calculating elastic constants in ABACUS.

**Comment by mohanchen** [comment]
I think this is a great idea, but you should clarify how many detailed steps we need to implement this new feature.

**Comment by mohanchen** [comment]
For these new features, I think we don't have enough time to implement it right now. However, I think it is good to keep this in mind, so I move the issue to discussion panel.
